test/reducers/presences_reducer_test.js

```diff
@@ -153,3 +153,27 @@ describe("presences reducer", () => {
     })
   })
 })
+
+describe("selectors", () => {
+  describe("findCurrentUser", () => {
+    const stubUser = { token: "123" }
+    const stubPresences = [stubUser]
```

Comment:
hey @laumontemayor and @qlaire! not a blocker, but it would make the test more robust to have another presence in `stubPresences`, but with a different `token` value, as that communicates that value is the identifying criteria